Cargo.toml

```diff
@@ -12,6 +12,7 @@ links = "rustls_ffi"
 rust-version = "1.64"
 
 [features]
+default = ["aws_lc_rs"]
```

Comment:
fwiw I think the rustls featuring naming was a mistake on my part, and it should've been `aws-lc-rs` all along. Not sure if we want to continue with that for consistency, or use hyphens from here on?